test/built-ins/Map/prototype/getOrInsertComputed/does-not-evaluate-callbackfn-if-key-present.js

```diff
@@ -13,7 +13,6 @@ info: |
   6. Let value be ? Call(callbackfn, undefined, « key »).
   ...
 features: [WeakMap, upsert]
-flags: [noStrict]
 ---*/
 var map = new Map([
   [1, 0]
```

Comment:
Please consider making `callback` increment increment a captured counter; this test doesn't preclude evaluating the callback but quashing the exception